.github/workflows/ci.yml

```diff
@@ -1,8 +1,15 @@
 ---
 name: "CI"
 on:  # yamllint disable
-  - "push"
-  - "pull_request"
+  push:
+  pull_request:
+    paths-ignore:
+      - "nautobot/docs/**"
+      - "examples/**"
```

Comment:
Changes to the ~dummy~ example plugin should probably still trigger CI, I think?